### PR TITLE
Fix Markdown of Heroku Link

### PR DESCRIPTION
**Pull request by nelsonic**
heroku-link-markdown-fail

image

This PR fixes that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
   
 
 <div align="center">
-**Try it at: [liveview-chat-example.herokuapp](https://liveview-chat-example.herokuapp.com/)**
+**Try it**: [**liveview-chat-example.herokuapp**](https://liveview-chat-example.herokuapp.com/)
```
Comment (nelsonic) [inline README.md:8]:
```suggestion
**Try it**: [**liveview-chat-example.herokuapp**]("https://liveview-chat-example.herokuapp.com")
```

**Comment by nelsonic** [comment]
Almost there:
image

**Comment by nelsonic** [comment]
https://github.com/dwyl/phoenix-liveview-chat-example/blob/cc7b2858d148aeb19a49754c64a8fe8fc914e71b/README.md
image

Link works now: https://liveview-chat-example.herokuapp.com/

image

Thanks again @SimonLab 🚀

**Comment by SimonLab** [comment]
Thanks for the review of the Previous PR @nelsonic ! And thanks for fixing the markdown!
I spotted this warning on the PR:
image
I'm not sure where it comes from but I'll have a look.

**Review by SimonLab — APPROVED** [review]
Thanks @nelsonic

**Pull request merged**